### ISSUE TITLE
KAS 1.0: Pretend the KAS link is a docking link

### ISSUE THREAD

**Issue by ihsoft** [ihsoft/KAS#237]
Some third-party mods rely on the docking modules to distinguish the vessel parts vs the docked vessels. Since in KAS the parts are "just connected", those mods cannot detect the KAS linked vessels as "docked".

Consider inheriting or simulating a docking module to keep these mods working. An investigation is needed to verify if it's even feasible.

**Comment by taniwha** [comment]
I don't think this is really feasible without undesirable consequences. It would be much better for those mods to look for KAS connectors as well as docking ports.
